spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -4,21 +4,33 @@
 
 RSpec.describe Reek::Spec::ShouldReekOf do
   describe 'smell type check' do
-    let(:src) { 'def double_thing() @other.thing.foo + @other.thing.foo end' }
+    let(:ruby) { 'def double_thing() @other.thing.foo + @other.thing.foo end' }
 
     it 'reports duplicate calls' do
-      expect(src).to reek_of(:Duplication)
+      expect(ruby).to reek_of(:Duplication)
     end
 
     it 'does not report any other smell types' do
-      expect(src).not_to reek_of(:FeatureEnvy)
+      expect(ruby).not_to reek_of(:FeatureEnvy)
+    end
+
+    it 'reports duplicate calls to @other.thing' do
+      expect(ruby).to reek_of(:Duplication, parameters: { name: '@other.thing' })
+    end
+
+    it 'reports duplicate calls to @other.thing.foo' do
+      expect(ruby).to reek_of(:Duplication, parameters: { name: '@other.thing.foo' })
+    end
+
+    it 'does not report any feature envy' do
```

Comment:
The example description is now misleading. It's not about FeatureEnvy, it's about "does not report any other smell types" like in the old example description -> let's change that back to what it was before.